web/portal/platform/src/entities/TerminalManufacturer/SelectOptions.ts

```diff
@@ -0,0 +1,27 @@
+import defaultEntityBehavior from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { SelectOptionsType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import store from 'store';
+
+const TerminalManufacturerSelectOptions: SelectOptionsType = ({
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const TerminalManufacturer = entities.TerminalManufacturer;
+
+  return defaultEntityBehavior.fetchFks(
+    TerminalManufacturer.path,
+    ['id'],
+    (data: any) => {
+      const options: any = {};
```

Comment:
use array options so that it can be sorted

```suggestion
      const options: DropdownChoices = [];
```